aspnetcore/fundamentals/configuration/index.md

```diff
@@ -217,7 +217,28 @@ setx Logging__1__Level=Information
 
 ### Environment variables set in launchSettings.json
 
-Environment variables set in *launchSettings.json* override those set in the system environment.
+Environment variables set in *launchSettings.json* override those set in the system environment. For example, the ASP.NET Core web templates generate a *launchSettings.json* file that sets the [Kestrel endpoint configuration](#kestrel) to:
+
+```json
+"applicationUrl": "https://localhost:5001;http://localhost:5000"
+```
+
+The preceding setting overrides values set in the environment.
```

Comment:
Configuring the "applicationUrl" sets/overrides the ASPNETCORE_URLS environment variable, but that feels like more of an implementation detail. I would expect to see something more like the following in a section named "Environment variables set in launchSettings.json":

```json
"environmentVariables": {
  "ASPNETCORE_ENVIRONMENT": "Development"
},
```